.travis.yml

```
@@ -0,0 +1,24 @@
+language: objective-c
```

Maybe we should use swift as language?

```suggestion
language: swift
```